data/Sri Guru Granth Sahib Ji/0196.json

```diff
@@ -403,7 +403,7 @@
         "translations": {
           "English": {
             "Dr. Sant Singh Khalsa": {
-              "translation": "The Guru, became my Help and Support. ||3||",
+              "translation": "were obtained when the Merciful Lord, the Guru, became my Help and Support. ||3||",
```

Comment:
```suggestion
              "translation": "when the Merciful Lord, the Guru, became my Help and Support. ||3||",
```

Reply:
Translation is fine here, `were obtained` belongs in this line. Cross-reference other translations and Gurbani line itself.